Robot/src/main/java/com/swrobotics/robot/commands/AutoAimCommand.java

```diff
@@ -0,0 +1,44 @@
+package com.swrobotics.robot.commands;
+
+import com.swrobotics.robot.subsystems.drive.DrivetrainSubsystem;
+import com.swrobotics.robot.subsystems.vision.Limelight;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class AutoAimCommand extends CommandBase {
+    private final Limelight limelight;
+    private final DrivetrainSubsystem drivetrainSubsystem;
+
+    Rotation2d limelight_target_angle;
+    Pose2d drivetrain_pose;
+
+    Rotation2d robot_target_angle;
+
+    public AutoAimCommand(Limelight limelight, DrivetrainSubsystem drivetrainSubsystem) {
+        this.limelight = limelight;
+        this.drivetrainSubsystem = drivetrainSubsystem;
+
+        addRequirements(limelight, drivetrainSubsystem);
+    }
+
+    @Override
+    public void initialize() {
+        // Get System State at Command Initialization to Ensure equal state
+        limelight_target_angle = limelight.getXAngle();
+        drivetrain_pose = drivetrainSubsystem.getPose();
+        robot_target_angle = drivetrain_pose.getRotation().plus(limelight_target_angle);
+    }
+
+    @Override
+    public void execute() {
+        drivetrainSubsystem.setTargetRotation(robot_target_angle);
+        drivetrainSubsystem.setTargetTranslation(drivetrain_pose.getTranslation(), false);
```

Comment:
You don't need to update the translation if you don't need to move, since you're only rotating here

Reply:
If we got pushed then we would miss, so it's good to store the full scoring position.

Reply:
The setTargetTranslation() is a little misleading, what it expects as input is essentially the same as the joystick input (i.e. [0, 0] to not move, [1, 0] to move right, [0, 1] to move forward). This command is only focusing on rotation, so this isn't needed here